src/core/define.ml

```diff
@@ -144,6 +145,7 @@ let infos = function
 	| EraseGenerics -> "erase_generics",("Erase generic classes on C#",[Platform Cs])
 	| EvalDebugger -> "eval_debugger",("Support debugger in macro/interp mode. Allows host:port value to open a socket. Implies eval_stack.",[])
 	| EvalStack -> "eval_stack",("Record stack information in macro/interp mode",[])
+	| EvalStackSize -> "eval_stack_size",("Set maximum call stack size for eval",[])
```

Comment:
Hm, shouldn't this have `[Platform Eval]` (and the other eval defines too for that matter)?

Reply:
+ might be good to mention what the default size is.

Reply:
Maybe. Wouldn't that seem confusing for users, which try to target js?

Reply:
I don't know, if they don't know what eval is, they would probably be confused about "eval-stack-size" anyway.